meerk40t/gui/wxmscene.py

```diff
@@ -1055,49 +1064,49 @@ def last_snap(self, value):
     def listen_make_ref(self, origin, *args):
         node = args[0]
         self.reference_object = node
-        self.context.signal("reference")
+        self.signal("reference")
 
     @signal_listener("draw_mode")
     def on_draw_mode(self, origin, *args):
         if self._tool_widget is not None:
             orgx = 5
             orgy = 5
             # Are guides drawn?
-            if self.context.draw_mode & DRAW_MODE_GUIDES == 0:
```

Comment:
This should read, `self.root.draw_mode` since drawmode is almost certainly in root.